esmvaltool/utils/recipe_test_workflow/doc/source/user_guide/workflow.rst

```diff
@@ -27,8 +27,9 @@ The |RTW| performs the following steps:
   :Executes:
      The ``configure.py`` script from the |Rose| app
   :Details:
-     Runs once at the start of the workflow, immediately after the successful
-     completion of the ``install_cold`` job
+     Runs once at the beginning of the workflow, after the install_cold task,
+     and then at the start of each cycle, after the successful completion
+     of the previous cycle's compare job
```

Comment:
Would it be possible to use the original documentation text as the start of this new sentence, please? (Since it formats the `install_code` text appropriately, and correctly uses the term "job" (a thing that runs) rather than "task" (a definition in a file).

Reply:
0d23b687079c2b3354afeac480ea58c115b68bbd